We already have the meson_python wheel, can't built it in isolated mode inside OBS

Index: meson_python-0.13.2/tests/conftest.py
===================================================================
--- meson_python-0.13.2.orig/tests/conftest.py
+++ meson_python-0.13.2/tests/conftest.py
@@ -156,18 +156,7 @@ def disable_pip_version_check():
 
 @pytest.fixture(scope='session')
 def pep518_wheelhouse(tmp_path_factory):
-    wheelhouse = os.fspath(tmp_path_factory.mktemp('wheelhouse'))
-    meson_python = os.fspath(package_dir.parent.parent)
-    # Populate wheelhouse with wheel for the following packages and
-    # their dependencies.  Wheels are downloaded from PyPI or built
-    # from the source distribution as needed.  Sources or wheels in
-    # the pip cache are used when available.
-    packages = [
-        meson_python,
-    ]
-    cmd = [sys.executable, '-m', 'pip', 'wheel', '--no-build-isolation', '--wheel-dir', wheelhouse, *packages]
-    subprocess.run(cmd, check=True)
-    return wheelhouse
+    return str(package_dir.parent.parent / 'build')
 
 
 @pytest.fixture
